author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_accessor :items
```

Comment:
It will be a good idea to be able to read the names (first_name and last_name) and the id of authors. Please make them readerable by setting `attr_reader`